modules/ROOT/pages/rtm-agent-proxy-config.adoc

```diff
@@ -57,6 +57,28 @@ This example configures the agent to work with a proxy server that requires auth
 amc_setup -H myToken myMuleServer -P acme.proxy.com 443 internalAdmin Ins1d3V0icePassword
 ----
 
+== Verify That the Proxy Server Does Not Modify the Agent Certificate 
```

Comment:
Verify That the Proxy Server Does Not Modify the **Runtime Manager Certificate**